api/URL.json

```diff
@@ -1018,10 +1018,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "6.1"
```

Comment:
`webkitURL` was added in Safari 6, and I'd expect `toString` to be there from the start. Could you look double check this with an instance / custom test?

Reply:
This has an instance test, actually, but it may need its own custom test.

Reply:
Hmm...actually, it looks like this data is correct.  `.toString()` in Safari 6 returns `[object URL]`, but in Safari 6.2 (6.1) it returns `https://mdn-bcd-collector.appspot.com` (given `new webkitURL('https://mdn-bcd-collector.appspot.com')`).

Reply:
Aha. Excellent find! Not that Safari 6 support matters any more, but this is pretty important issue, since `new webkitURL(url).toString()` is a convenience for normalizing a URL that wouldn't work then.